### ISSUE TITLE
Add search offline players by name in chat channel

### ISSUE THREAD

**Issue by Marc-Spector** [FAForever/downlords-faf-client#2700]

**Comment by Sheikah45** [comment]
I don't think it should go in the chat channel because the chat channel is for chatting not for offline players

**Comment by Marc-Spector** [comment]
Is there a better place?

**Comment by Marc-Spector** [comment]
Some players complain that when a player goes offline, we cannot interact with him, for example, add the player to the black list, update notes.

**Comment by Marc-Spector** [comment]
The only solution is to go to the replay page and interact with it.

**Comment by Marc-Spector** [comment]
How about making a new "Players" tab?

**Comment by Sheikah45** [comment]
I would just make it some menu option that pops up a new window
